lib/Sema/TypeChecker.h

```diff
@@ -1124,6 +1124,15 @@ class TypeChecker final : public LazyResolver {
   /// \returns true if \c t1 is a subtype of \c t2.
   bool isSubtypeOf(Type t1, Type t2, DeclContext *dc);
 
+  /// \brief Determine whether one type is a subclass of another.
+  ///
+  /// \param t1 The potential subtype.
+  /// \param t2 The potential supertype.
+  /// \param dc The context of the check.
+  ///
+  /// \returns true if \c t1 is a subtype of \c t2.
```

Comment:
I'll make a patch to update this on master tomorrow, as well as clarifying the FIXME comment below.